### DISCUSSION TITLE
Safety of remote file access?

### DISCUSSION THREAD

**Discussion by pmetzger** [zed-industries/zed#6571] [Feature Requests]
Howdy! Zed seems really cool, but it warns me that having a remote collaborator implies that they get unrestricted access to my local file system. What are the plans for improving the safety of remote collaboration? I'm especially concerned that I don't want to overshare with the server either.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
